nullaway/src/test/java/com/uber/nullaway/NullAwayCoreTests.java

```diff
@@ -709,4 +710,40 @@ public void staticCallZeroArgsNullCheck() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void allowLibraryModelsOverrideAnnotations() {
+    makeTestHelperWithArgs(
+            Arrays.asList(
+                "-processorpath",
+                TestLibraryModels.class
+                    .getProtectionDomain()
+                    .getCodeSource()
+                    .getLocation()
+                    .getPath(),
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:AllowLibraryModelsOverrideAnnotations=true"))
```

Comment:
Worth adding the converse test. A test case showing that, when `XepOpt:NullAway:AllowLibraryModelsOverrideAnnotations` is not set (or set to false, if you think we might change the default), the library model for `com.uber.Foo` is still ignored.

Reply:
9b141bc393d6bba7b62760251cf2999ac9bb5e16